arrows/vxl/convert_image.cxx

```diff
@@ -459,6 +459,10 @@ ::filter( kwiver::vital::image_container_sptr image_data )
       return image_data;                                                      \
     }                                                                         \
     vil_image_view< ipix_t > input = d->apply_transforms< ipix_t >( view );   \
+    if( d->format == "copy" )                                                 \
+    {                                                                         \
+      return std::make_shared< vxl::image_container >( input );               \
```

Comment:
```suggestion
       vil_image_view< ipix_t > output = d->scale_and_convert< ipix_t >( input ); \
       return std::make_shared< vxl::image_container >( output  );                        \
```
With `auto` used if desired.

Reply:
Done. I also made some tweaks to ``scale_and_convert` so that the case of no scaling is not pessimized.